libsoftwarecontainer/src/gateway/cgroups/cgroupsgateway.h

```diff
@@ -112,11 +112,24 @@ class JSonError : public CgroupsGatewayError
     }
 };
 
+class InvalidValueError : public CgroupsGatewayError
+{
```

Comment:
Yes, I was thinking of this also. Couldn't decide. I guess the error text describes in more detail anyway.